src/components/UserInfo.js

```diff
@@ -1,16 +1,27 @@
+import { api } from "../pages/index.js";
+
 class UserInfo {
-  constructor(nameSelector, jobSelector) {
-    this._name = document.querySelector(nameSelector);
-    this._job = document.querySelector(jobSelector);
+  constructor(userData) {
+    this._name = document.querySelector(".profile__name");
+    this._name.textContent = userData.name;
+    this._about = document.querySelector(".profile__about");
+    this._about.textContent = userData.about;
+    this._avatar = document.querySelector(".profile__image");
+    this._avatar.src = userData.avatar;
   }
 
   getUserInfo() {
-    return { name: this._name.textContent, job: this._job.textContent };
+    return api.getUserData();
```

Comment:
```js

  getUserInfo() {
    return { name: this._name.textContent, job: this._job.textContent };
```

This method should be the same which you had in the 8th sprint

You don't need to fetch the data which you have in the profile on the page

Reply:
fixed this